StyleCop.Analyzers/StyleCop.Analyzers/SpacingRules/SA1003SymbolsMustBeSpacedCorrectly.cs

```diff
@@ -268,15 +267,12 @@ private static void HandlePrefixUnaryExpression(SyntaxNodeAnalysisContext contex
             switch (unaryExpression.OperatorToken.Kind())
             {
             case SyntaxKind.PlusToken:
-                analyze = context.IsAnalyzerSuppressed(SA1022PositiveSignsMustBeSpacedCorrectly.DiagnosticId);
-                break;
             case SyntaxKind.MinusToken:
-                analyze = context.IsAnalyzerSuppressed(SA1021NegativeSignsMustBeSpacedCorrectly.DiagnosticId);
-                break;
             case SyntaxKind.PlusPlusToken:
             case SyntaxKind.MinusMinusToken:
-                analyze = context.IsAnalyzerSuppressed(SA1020IncrementDecrementSymbolsMustBeSpacedCorrectly.DiagnosticId);
+                analyze = false;
                 break;
```

Comment:
:thought_balloon: This could do with some comments indicating what rule is handling these cases. It will improve readability for people without in depth knowledge